gorm.go

```diff
@@ -164,6 +164,80 @@ func Open(dialector Dialector, config *Config) (db *DB, err error) {
 	return
 }
 
+// DefaultConfig default config.
+func DefaultConfig() *Config {
+	c := &Config{
+		NamingStrategy: schema.NamingStrategy{},
+		Logger:         logger.Default,
+		NowFunc:        func() time.Time { return time.Now().Local() },
+		ClauseBuilders: map[string]clause.ClauseBuilder{},
+		Plugins:        map[string]Plugin{},
+		cacheStore:     &sync.Map{},
+	}
+
+	return c
+}
+
+// Apply apply option for Config.
+func (config *Config) Apply(opts ...ConfigOption) {
+	for _, o := range opts {
+		o(config)
+	}
+}
+
+// OpenWithOption create *db.DB with ConfigOption
+// return *db.DB,error.
+func OpenWithOption(dialector Dialector,
```

Comment:
Maybe name this `OpenWithOptions` (plural) since it accepts multiple option arguments?

Reply:
Using OpenWithOptions naming, it is indeed relatively better, thank you for your suggestions.